scripts/move.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+pushd $(dirname "$0")/..
+
+export RPC_URL="http://localhost:5050";
+
+export WORLD_ADDRESS=$(cat ./manifests/dev/manifest.json | jq -r '.world.address')
+
+export ACTIONS_ADDRESS=$(cat ./manifests/dev/manifest.json | jq -r '.contracts[] | select(.name == "dojo_starter::systems::actions::actions" ).address')
```

Comment:
We don't actually need to grep the address. We can directly provide the name of the contract. Which is also a nice example of how sozo works with contract name.

```
sozo execute --world <> dojo_starter::systems::actions::actions move -c 1 --wait
```

Reply:
Fixed on commit [fa926a9](https://github.com/dojoengine/dojo-starter/pull/52/commits/fa926a95260b5446e67be567ce2695f3b2ced8cc). Please check